### PR TITLE
Finished MVP 1 of the workshop

### PR DESCRIPTION
**Pull request by jonleung**

### CODE REVIEW

**Comment by jonleung** [comment]
I would love some feedback on this @gemmabusoni @ssun098 https://github.com/hackedu/hack-camp/blob/4634358eeccf2aa17d8c87da16844b0bee3c1219/cohort_4/playbook/ACTIVITIES.md#instagram

**Comment by jonleung** [comment]
@zachlatta @max @gemmabusoni @ssun098 I think it may be a bit too long...

**Comment by zachlatta** [comment]
By the way @jonleung, you can comment directly line-by-line (https://help.github.com/articles/commenting-on-the-diff-of-a-pull-request/).

I'm happy to go ahead and merge this. I'm seeing some formatting mistakes, that I'm also happy to fix while I'm merging it (I can just cherry pick the commits over to master and modify them while I'm doing that).

**Comment by jonleung** [comment]
Closes #185

File: cohort_4/playbook/workshops/README.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
-# Workshops
-
-This directory contains all of the workshops for Hack Camp.
-
-A list of the workshops, in the order to run them. Refer to
-[`SCHEDULE.md`](../SCHEDULE.md) for where these fit into the overall schedule.
-
-1. [Portfolio](./portfolio/) ([facilitation](../ACTIVITIES.md#portfolio)) -
-   Introduces HTML & CSS
```
Comment (zachlatta) [inline cohort_4/playbook/workshops/README.md:9]: Hmm, I probably shouldn't have merged this.

Why was this file deleted?

Comment (jonleung) [inline cohort_4/playbook/workshops/README.md:9]: Hm, not sure how that happened, I'll try to be more careful in the future!
